package.json

```diff
@@ -24,5 +24,11 @@
     "ts-node": "^9.1.1",
     "typescript": "^4.2.2"
   },
-  "dependencies": {}
+  "dependencies": {
+    "boxen": "^5.0.0",
+    "chalk": "^4.1.0",
+    "figlet": "^1.5.0",
+    "inquirer": "^8.0.0",
+    "yargs": "^16.2.0"
+  }
 }
```

Comment:
A nice-to-have: let's add a yarn script for compiling and running. (Note there's an existing `dev` script 🙂)
```
"prod": "tsc -p . && node bin/index.js"
```